.gitignore

```diff
@@ -112,3 +112,6 @@ venv.bak/
 auto-save-list
 tramp
 .\#*
+
+# VScode
+.vscode
```

Comment:
IMHO this doesn't belong into this PR. I suggest to use `git rebase` to remove the last two commits.